saturn-datepicker/src/datepicker/month-view.ts

```diff
@@ -226,7 +227,7 @@ export class SatMonthView<D> implements AfterContentInit {
   }
 
   /** Handles keydown events on the calendar body when calendar is in month view. */
-  _handleCalendarBodyKeydown(event: KeyboardEvent): void {
+  @HostListener('document:keydown', ['$event']) _handleCalendarBodyKeydown(event: KeyboardEvent): void {
```

Comment:
and don't forget to return this back

Reply:
https://github.com/SaturnTeam/saturn-datepicker/pull/57/commits/f618d69c5275f26a68291f1410b743bb5bec5264